Project Improvement

fridorin opened the pull request:
Improved user experience in using the application. Fixing bugs in the code. Also added domains and updated binaries

fridorin committed:
Update binaries

Update from bol-van/zapret/releases/tag/v70.1

fridorin committed:
Enhancements

The user is now guaranteed to get the current version information without complications

fridorin committed:
fix

Sorry, the perfectionist was in pain)

fridorin committed:
Added domains

fridorin committed:
Enhancements

More elegant start-up

fridorin committed:
Enhancements

A more elegant stop

fridorin committed:
fix too

fridorin committed:
hot fix

(

Flowseal commented on ipset-discord.txt:
```diff
@@ -719,4 +719,3 @@
 188.114.99.224
 204.11.56.48
 209.85.233.207
-34.126.226.51
```
Есть какие-то причины удалять?
https://github.com/Flowseal/zapret-discord-youtube/issues/1103#issuecomment-2629224566

fridorin committed:
Update ipset-discord.txt

fridorin committed:
fix

fridorin committed:
Update list-general.txt

fridorin commented:
Исправил всё, кроме поддоменов. Почти не работал с pull request, но, вроде, всё сделал правильно

fridorin commented:
@Flowseal ну так что?

Flowseal commented:
> @Flowseal ну так что?

Как время будет, посмотрю

amozebus commented on service_install.bat:
```diff
@@ -1,23 +1,27 @@
-if "%arg%" == "admin" (
-    echo Restarted with admin rights
-) else (
+:: Admin rights check
```
См. #1358

amozebus commented on service_install.bat:
```diff
@@ -1,23 +1,27 @@
-    echo Restarted with admin rights
-) else (
+:: Admin rights check
+net session >nul 2>&1
```
См. #1358

amozebus commented on service_install.bat:
```diff
@@ -1,23 +1,27 @@
+:: Admin rights check
+net session >nul 2>&1
+if %errorLevel% neq 0 (
+    echo Requesting admin rights...
```
См. #1358

fridorin committed:
Update winws.exe

Обновлен bin/winws.exe из zapret-win-bundle/master

fridorin committed:
downgrade admin rights check

вернул все обратно из за https://github.com/Flowseal/zapret-discord-youtube/pull/1358

Flowseal committed:
Fix for undefined version

Flowseal committed:
Removed message on service deleting

As we can't guarantee that services were removed successfully

Flowseal committed:
Removed message on service installing

As we can't guarantee that services were launched successfully

Flowseal committed:
Update list-discord.txt

Flowseal committed:
Update list-general.txt

amozebus commented:
@Flowseal Стоит выпустить новый релиз после обновления `winws.exe`
